main/webapp/modules/core/MOD-INF/controller.js

```diff
@@ -223,6 +223,8 @@ function registerImporting() {
 
   IM.registerFormat("text/xml", "XML files", "XmlParserUI", new Packages.com.google.refine.importers.XmlImporter());
   IM.registerFormat("binary/text/xml/xls/xlsx", "Excel files", "ExcelParserUI", new Packages.com.google.refine.importers.ExcelImporter());
+  IM.registerFormat("application/vnd.ms-excel", "Excel files", "ExcelParserUI", new Packages.com.google.refine.importers.ExcelImporter());
+  IM.registerFormat("application/vnd.openxmlformats-officedocument.spreadsheetml.*", "Excel files", "ExcelParserUI", new Packages.com.google.refine.importers.ExcelImporter());
```

Comment:
This should be removed - we should only need to add a mapping from the MIME type to the internal format identifier, with `IM.registerMimeType` (see below).